src/quacc/recipes/espresso/phonons.py

```diff
@@ -39,28 +39,39 @@ class PhononDosSchema(TypedDict):
 
 @job
 def phonon_job(
-    copy_files: SourceDirectory | dict[SourceDirectory, Filenames],
+    copy_files: (
+        SourceDirectory | list[SourceDirectory] | dict[SourceDirectory, Filenames]
+    ),
     parallel_info: dict[str] | None = None,
     test_run: bool = False,
+    is_phcg: bool = False,
     **calc_kwargs,
 ) -> RunSchema:
     """
     Function to carry out a basic ph.x calculation. It should allow you to
     use all the features of the [ph.x binary](https://www.quantum-espresso.org/Doc/INPUT_PH.html)
 
-    This job requires the results of a previous pw.x calculation, you might
-    want to create your own flow to run both jobs in sequence.
+    `ph.x` calculates the dynamical matrix at a set of q-points within the Density
+    Functional Perturbation Theory (DFPT) framework. The dynamical matrix is used to calculate the phonon frequencies and eigenvectors. Various other properties can be
+    calculated using other post-processing tools.
 
     Parameters
     ----------
     copy_files
-        Files to copy (and decompress) from source to the runtime directory.
+        Source directory or directories to copy files from. If a `SourceDirectory` or a
+        list of `SourceDirectory` is provided, this interface will automatically guess
+        which files have to be copied over by looking at the binary and `input_data`.
+        If a dict is provided, the mode is manual, keys are source directories and values
+        are relative path to files or directories to copy. Glob patterns are supported.
     parallel_info
         Dictionary containing information about the parallelization of the
         calculation. See the ASE documentation for more information.
     test_run
         If True, a test run is performed to check that the calculation input_data is correct or
         to generate some files/info if needed.
+    is_phcg
```

Comment:
Is there anything more user-friendly we can have here, or is this known by the Espresso users? I don't know what "phcg" is even based on the docstring. 

Edit: Ah, I see it's a binary. It seems to me that "is" may not be appropriate here. Perhaps `use_phcg` makes more sense?